api/python/quilt3/packages.py

```diff
@@ -521,7 +521,7 @@ def browse(cls, name, registry=None, top_hash=None):
         return cls._browse(name=name, registry=registry, top_hash=top_hash)
 
     @classmethod
-    def _browse(cls, name, registry, top_hash):
+    def _browse(cls, name, registry=None, top_hash=None):
```

Comment:
Is this change needed for the fix?

Reply:
Not in this context, but would also break execution [here](https://github.com/quiltdata/quilt/blob/c0dd45ac6d47d21345991690ee042f3d00c25787/api/python/quilt3/imports.py#L49) without a default argument. Similar to the issue with the required argument.